### ISSUE TITLE
[Bug] Visual Tag Builder node will infinity long in ComfyUI nodes 2.0

### ISSUE THREAD

**Issue by truelife2** [diodiogod/TTS-Audio-Suite#344]
**Describe the bug**
Visual Tag Builder node 's height will inifinity long in ComfyUI nodes 2.0

**Init log**
[INFO] Total VRAM 12287 MB, total RAM 32637 MB
[INFO] pytorch version: 2.13.0+cu130
[INFO] Set vram state to: NORMAL_VRAM
[INFO] Device: cuda:0 NVIDIA GeForce RTX 3060 : cudaMallocAsync
[INFO] Using async weight offloading with 2 streams
[INFO] Enabled pinned memory 13054.0
[INFO] Using pytorch attention
[INFO] aimdo: src-win/cuda-detour.c:38:INFO:aimdo_setup_hooks: installing 6 hooks
[INFO] aimdo: src/control.c:262:INFO:comfy-aimdo NVML pressure enabled
[INFO] aimdo: src-win/shmem-detect.c:80:INFO:comfy-aimdo WDDM adapter match: NVIDIA GeForce RTX 3060 runtime_luid=00000000:00010d5b dxgi_luid=00000000:00010d5b
[INFO] aimdo: src/control.c:277:INFO:comfy-aimdo inited for GPU: NVIDIA GeForce RTX 3060 (VRAM: 12287 MB)
[INFO] DynamicVRAM support detected and enabled
[INFO] Python version: 3.13.12 (tags/v3.13.12:1cbe481, Feb  3 2026, 18:22:25) [MSC v.1944 64 bit (AMD64)]
[INFO] ComfyUI version: 0.31.0
[INFO] comfy-aimdo version: 0.4.13
[INFO] comfy-kitchen version: 0.2.28
[INFO] comfyui-frontend-package version: 1.48.7
[INFO] comfyui-workflow-templates version: 0.11.37
[INFO] comfyui-embedded-docs version: 0.5.9
[INFO] comfy-kitchen version: 0.2.28
[INFO] comfy-aimdo version: 0.4.13
[INFO] [Prompt Server] web root: C:\ai\ComfyUI_windows_portable\python_embeded\Lib\site-packages\comfyui_frontend_package\static

ℹ️ Critical package versions: NumPy 2.5.1, Librosa 0.11.0, Numba 0.65.1, PyTorch 2.13.0+cu130, TorchAudio 2.11.0+cu130, Transformers 5.14.1, Accelerate 1.14.0, SoundFile 0.14.0
⚠️ Failed to import Qwen3-TTS: Numba needs NumPy 2.4 or less. Got NumPy 2.5.
   Make sure official implementation is in C:\ai\ComfyUI_windows_portable\ComfyUI\custom_nodes\TTS-Audio-Suite\engines\qwen3_tts\impl
======================================================================
🚀 TTS Audio Suite v5.6.5
Universal multi-engine TTS extension for ComfyUI
🐍 Python 3.13.12 detected
✅ TTS Audio Suite v5.6.5 loaded with 56 nodes:
   • ⚙️ ChatterBox Official 23-Lang Engine
   • ⚙️ ChatterBox TTS Engine
   • ⚙️ CosyVoice3 Engine
   • ⚙️ Dots TTS Engine
   • ⚙️ DramaBox Engine
   • ⚙️ Echo-TTS Engine
   • ⚙️ F5 TTS Engine
   • ⚙️ Fish Audio S2 Pro Engine
   • ⚙️ Granite ASR Engine
   • ⚙️ Higgs Audio 2 Engine
   • ⚙️ Higgs Audio v3 Engine
   • ⚙️ IndexTTS-2 Engine
   • ⚙️ MOSS SoundEffect v2 Engine
   • ⚙️ MOSS-TTS Engine
   • ⚙️ OmniVoice Engine
   • ⚙️ Qwen3-TTS Engine
   • ⚙️ RVC Engine
   • ⚙️ Step Audio EditX Engine
   • ⚙️ VibeVoice Engine
   • ♻️ Refresh Voice Cache
   • ✏️ ASR Transcribe
   • 🌈 IndexTTS-2 Emotion Vectors
   • 🌈 IndexTTS-2 Text Emotion
   • 🌊 Audio Wave Analyzer
   • 🌩️ Sound Effects
   • 🎓 Model Training
   • 🎙️ Voice Capture
   • 🎛️ MOSS Training Config
   • 🎛️ RVC Training Config
   • 🎞️ MOSS Clip Staging
   • 🎤 TTS Text
   • 🎨 Step Audio EditX - Audio Editor
   • 🎨 Voice Designer
   • 🎭 Character Voices
   • 🎭 Load RVC Character Model
   • 🏷️ Multiline TTS Tag Editor
   • 👄 F5-TTS Speech Editor
   • 💾 Save Character Voice
   • 📐 Visual Tag Builder
   • 📝 ASR Punctuation / Truecase
   • 📝 Phoneme Text Normalizer
   • 📦 MOSS Dataset Prep
   • 📦 RVC Dataset Prep
   • 📺 TTS SRT
   • 📺 Text to SRT Builder
   • 🔄 Voice Changer
   • 🔧 Audio Analyzer Options
   • 🔧 F5-TTS Edit Options
   • 🔧 RVC Pitch Extraction Options
   • 🔧 SRT Advanced Options
   • 🔧 Viseme Mouth Shape Options
   • 🗣️ Silent Speech Analyzer
   • 🤐 Noise or Vocal Removal
   • 🤐 Voice Fixer
   • 🥪 Merge Audio
   • 🧾 MOSS Dataset Rows
======================================================================

**Provide your ERROR full log**
drag drop the node without has any log in cmd window.

**Screenshots**

<img width="839" height="620" alt="Image" src="https://github.com/user-attachments/assets/d27381f8-b452-47b4-944a-7f0d0b05a89b" />

**System information**
 - OS: windows 11
 - ComfyUI type: portable web comfyui
 - Python version: 🐍 Python 3.13.12 detected

**Additional context**
weather draging or dropped , the height will inifinity long, until i delete the node

**Comment by diodiogod** [comment]
I don't currently support nodes 2.0. I hopefully will someday, but not planned right now.

**Comment by truelife2** [comment]
ok, thx your help, seem very people don't like nodes2.0.

but your integrating tts very usable, thx your share and may jesus bless you.
